platform-coop-toolkit/coop-map-directory-index
# feat: remove tool sectors

## Description

This PR removes the `sectors` field from the Tools model as this data will be determined by a tool's relationship to cooperatives.

## Steps to test

1. Apply the migration.
2. Edit a tool via the admin interface.

**Expected behavior:** The sectors field is no longer shown.

## Additional information

Not applicable.

## Related issues

Depends on #59.
